blog/releases/announcing-knative-v0-4-release.html

```diff
@@ -0,0 +1,15 @@
r markup--p-anchor" rel="nofollow noopener" target="_blank">Build</a>, and <a href="https://github.com/knative/eventing/releases/tag/v0.4.0" data-href="https://github.com/knative/eventing/releases/tag/v0.4.0" class="markup--anchor markup--p-anchor" rel="nofollow noopener" target="_blank">Eventing</a> repositories.</p><h3 name="363c" id="363c" class="graf graf--h3 graf-after--p">Learn more</h3><ul class="postList"><li name="c00d" id="c00d" class="graf graf--li graf-after--h3"><a href="https://github.com/knative/docs#welcome-to-knative" data-href="https://github.com/knative/docs#welcome-to-knative" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Welcome to Knative</a></li><li name="d88a" id="d88a" class="graf graf--li graf-after--li"><a href="https://github.com/knative/docs#documentation" data-href="https://github.com/knative/docs#documentation" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Getting started documentation</a></li><li name="a9ca" id="a9ca" class="graf graf--li graf-after--li"><a href="https://github.com/knative/docs#samples-and-demos" data-href="https://github.com/knative/docs#samples-and-demos" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Samples and demos</a></li><li name="fe75" id="fe75" class="graf graf--li graf-after--li"><a href="https://www.knative.dev/docs/community/README.md#meetings-and-work-groups" data-href="https://www.knative.dev/docs/community/README.md#meetings-and-work-groups" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Knative meetings and work groups</a></li><li name="cf03" id="cf03" class="graf graf--li graf-after--li"><a href="https://www.knative.dev/docs/community/README.md#questions-and-issues" data-href="https://www.knative.dev/docs/community/README.md#questions-and-issues" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Questions and issues</a></li><li name="19ce" id="19ce" class="graf graf--li graf-after--li">Knative on Twitter (<a href="https://twitter.com/KnativeProject" data-href="https://twitter.com/KnativeProject" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">@KnativeProject</a>)</li><li name="5fcb" id="5fcb" class="graf graf--li graf-after--li">Knative on <a href="https://stackoverflow.com/questions/tagged/knative" data-href="https://stackoverflow.com/questions/tagged/knative" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">StackOverflow</a></li><li name="3813" id="3813" class="graf graf--li graf-after--li graf--trailing">Knative <a href="https://slack.knative.dev/" data-href="https://slack.knative.dev/" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Slack</a></li></ul></div></div></section>
```

Comment:
This has some of the same link issues as #1072  -- some links to https://github.com/knative/docs
that should probably not point there.